cluster/common.sh

```diff
@@ -1124,13 +1145,18 @@ function generate-aggregator-certs {
     ./easyrsa --dn-mode=org \
       --req-cn=proxy-clientcfg --req-org=system:aggregator \
       --req-c= --req-st= --req-city= --req-email= --req-ou= \
-      build-client-full proxy-clientcfg nopass) &>${cert_create_debug_output} || {
-    # If there was an error in the subshell, just die.
-    # TODO(roberthbailey): add better error handling here
+      build-client-full proxy-clientcfg nopass) &>${cert_create_debug_output} || true
+
+  if [ ! -s  "${AGGREGATOR_CERT_DIR}/pki/private/ca.key" \
```

Comment:
Use [[ instead of [. Also consider looping to return a better error message.

https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B

Reply:
Ack. I'll change this to a loop.